### ISSUE TITLE
Blocks in RTE content doesn't gets updated when you use the clipboard to copy

### ISSUE THREAD

**Issue by jericyuen** [umbraco/Umbraco-CMS#16663]
### Which Umbraco version are you using? (Please write the *exact* version, example: 10.1.0)

13.3.2

### Bug summary

We are using a Block List to allow us to add multiple modules in the content section which allows me to add a Text Block module (that uses RTE editor).
Within that RTE we had enabled the Blocks in RTE as per below
https://docs.umbraco.com/umbraco-cms/fundamentals/backoffice/property-editors/built-in-umbraco-property-editors/rich-text-editor/rte-blocks

So we can add Blocks to allow us to add like features in the RTE which works perfectly fine.

We realised that when we use the clipboard button to duplicate a content
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/edb98979-b372-470c-8588-411a4bee58ec)

So when we use the clipboard to copy the Module content.
Any content from Block List added inside the RTE which has been copied, sort of doesn't gets updated. 
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/b611e0d1-58be-445e-937c-bac1570e8874)

So for example, we copy Block A and then use the clipboard to create Block B
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/33d1c2c3-1899-4065-a303-ade01719a5cc)

we'll have two blocks now. And if we update the Rich Text Row content (which is a Block list added inside RTE) within Block A, and we leave Block B's Rich Text Row as it is. Umbraco will keep returning back the same Content of the Blocks in RTE for both Block A and Block B. 

We've tried to change the content of Rich Text Row in Block B but it never gets updated in the front end at all

### Specifics

Tested in chrome browser and edge browser and it's the same

### Steps to reproduce

Install a vanilla umbraco 13.3.2, I've then installed Paul Seal's Clean starter kit just so we have some dummy contents and dummy block list
https://our.umbraco.com/packages/starter-kits/clean-starter-kit/

Go into Settings > Data Type > Richtext Editor > Enable Block from the settings
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/bc1e8b26-13a7-4551-9a67-284155044033)

Scroll down further and we add 'Rich Text Row' as the available Blocks
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/2cf9d57a-28df-443d-8829-bd22e355cf1f)

Go back to the Content section. I've used the Features page. 
In the Content Rows properties, I've removed all the modules and leave just one remaining Rich text row module
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/daa42dee-fe95-493f-9862-4e6fb985f40d)

We then click on the module, click on the Block Icon to add a Rich text row at the bottom of the module
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/89f9ee4a-0e1e-4f17-8ba1-2d73a992ae3e)

You can add some dummy text in it
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/60d4d60b-8d39-4c6f-9aa8-20acfd214ddf)

Go to your VS Solution and add the richTextRow.cshtml inside the Views/Partials/richtext/components/ folder
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/b4f5be24-347d-4d08-9453-01b2356740f1)

The code inside the richTextRow.cshtml will be as below

```
@inherits Umbraco.Cms.Web.Common.Views.UmbracoViewPage<Umbraco.Cms.Core.Models.Blocks.RichTextBlockItem>

@Model.Content.Value("content")
```

Now go back to the Content Section. Click on to duplicate the Rich Text Row module
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/cc8d99f1-2bfc-4a5a-a063-066b701da878)

Now you will have two blocks
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/d7417bca-6855-4a0d-82d1-cceba60c8287)

Load the front end of the page, which you'll see it's rendered out with the inner RTE module too
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/e24f9ea8-5940-49cc-a55a-8eb920b07e04)

Now go back to Content Section and on the first RTE module. And amend the Rich Text Row inside the RTE
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/6937203e-cfcc-4ecf-94b9-29765ad0a056)

You can leave the second RTE module as it is. Save and publish

And now view the Front End of the site

You can see that the bug is there showing both module content inside Blocks in Rich Text as below which is wrong
![image](https://github.com/umbraco/Umbraco-CMS/assets/1864036/e3bcd6f6-3fdd-40f4-b3e2-293e7c32967c)

### Expected result / actual result

Expected results should be. Both Blocks in Rich Text should show different content cos they are both individual blocks. But Umbraco thinks it's the same when you use the Clipboard features

---
_This item has been added to our backlog AB#42802_

**Comment by jericyuen** [comment]
Any updates on this? thanks

**Comment by NguyenThuyLan** [comment]
Thank you @jericyuen for your very detailed issue description. I am reproducing your issue.

**Comment by jericyuen** [comment]
Just checking if there will be a fix soon on this 🙏

**Comment by jericyuen** [comment]
Hi @NguyenThuyLan  Just checking to see if there's any news on this? Thanks

**Comment by jericyuen** [comment]
Hi @NguyenThuyLan just checking on the update, as the client is chasing on this. It's been a month. Did you manage to figure things out? Thanks

**Comment by NguyenThuyLan** [comment]
Hi @jericyuen , sorry for the late response, I have informed the team about this issue. 😥, I will let you know soon

**Comment by leekelleher** [comment]
Fixed in #16866 for v13.5.0.
